fluent/handler.py

```diff
@@ -51,14 +51,14 @@ def format(self, record):
         data = dict([(key, value % record.__dict__)
                      for key, value in self._fmt_dict.items()])
 
-        self._structuring(data, record.msg)
+        self._structuring(data, record.msg, record.args)
```

Comment:
Hi,
`record` have `getMessage()` method for that: https://docs.python.org/2/library/logging.html#logging.LogRecord.getMessage

Something like this seems better:

    msg = record.getMessage()
    self._structuring(data, msg)

I need this feature, if you want i can make a PR.
Thanks